docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:22
```

Comment:
Let's pin to a full version and not a channel. This will allow us to harden the image.